README.md

```diff
@@ -2,3 +2,79 @@
 
 This repository contains examples in various languages that demonstrate how to
 build and test software with the EngFlow Remote Execution service.
+
+## Local execution
+### Building locally
+
+- Build each example with  
+
+  `bazel build //${DIRECTORY}/...`  
+
+  for `cpp`,  `csharp`, `docker`, `genrules`, `go`, `java`, `kotlin`, `perl`, `python`, `scala` and `typescript` or build them all with  
+  
+  `bazel build //...`
+- Build `swift` example with  
+  
+  `bazel build //swift:test --config=clang` 
+  
+  make sure `clang` is in your `PATH`.
+
+### Testing locally
+
+- Test each example with
+
+  `bazel test //${DIRECTORY}/...`  
+
+  for `java`, `scala` and `typescript`.
+
+- Test `swift` example with
+
+  `bazel test //swift:test --config=clang`
+
+- Test `python` example with
+
+  `bazel run //python:requirements.update && bazel test //python/...`
```

Comment:
Maybe have a prerequisites section and mention `bazel run //python:requirements.update` there (also `clang` for Swift)?

Reply:
In fact, I would like to have just few more details about the python sharding. Can we better have a readme file in the python directory? Can you write down some details about that and about its requirements?

Reply:
If we aren't going to have README files in all or most of the subdirectories, then maybe it's better to have the information here (because users won't know to look specifically under `python/` for its README). What do you think? I can write the Python part for sure, but let's settle on the structure, and then I'll send a PR for that.

Reply:
Lets do as you say. But at some point I think it example can evolve to include its own readme. Right now we have more details about `//java/com/engflow/notificationqueue/` maybe we need to improve visibility of that too